zoo-css-challenge/style.css

```
@@ -118,22 +140,38 @@ footer {
 }
 
 .giraffe {
-  background-color: #465775;
+  background-color: #483c46 /* #465775*/;
   color: #56e39f;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+  color: #f4743b /*#ef6f6c*/;
+}
+#strong {
+  font-size: 20px;
```

```suggestion
  font-size: 1.25rem;
```
We should use relative units like `rem` for `font-size`